scripts/test_examples.sh

```diff
@@ -201,44 +201,457 @@ stopKafka
+  -messageCount=\"\": Number of messages which will be send. Set to '0' to just watch
+  -microservice-label=\"vpp1\": microservice label; also set via 'MICROSERVICE_LABEL' env variable.
+  -offsetMsg=\"\": Use 'latest', 'oldest' or exact number of message offset
+")
+
+cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -h"
+testOutput "${cmd}" "${expected}"
+
+# Let us test the running without parameters - in example are generated 10 Kafka Messages to both topics but the consumed is only 5 messages from each topic beginning with offset 5
+expected=("Invalid partition -1 - defaulting to 0
+offset arg not set, using default value
+messageCount arg not set, using default value
+Offset: 0, message count: 10
+Started Kafka sync event handler...
+Started Kafka async event handler...
+All plugins initialized successfully
+Sending 10 sync Kafka notifications (protobuf) ...
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '6', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '7', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '8', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Sending 10 async Kafka notifications (protobuf) ...
+Async message successfully delivered
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '6', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '7', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '8', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key', 
 ")
 
 cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf"
 testOutput "${cmd}" "${expected}"
 
+# Let us test - in example are generated next more 10 Kafka Messages to both topics
+expected=("Invalid partition -1 - defaulting to 0
+offset arg not set, using default value
+messageCount arg not set, using default value
+Offset: 0, message count: 10
+Started Kafka sync event handler...
+Started Kafka async event handler...
+All plugins initialized successfully
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '6', key: 'async-proto-key', 
```

Comment:
keep it short, 
Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key',
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key',
will be enough